tests/test_sweet.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from astropy import units as u
+from astropy.utils.data import download_file
+from lightkurve.search import open as lc_read  # Will be read in 2.x
+from numpy.testing import assert_allclose
+
+from exovetter.tce import TCE
+from exovetter.vetters import Sweet
+
+
+@pytest.mark.remote_data
+def test_kepler_1026032():
+    filename = download_file('http://archive.stsci.edu/missions/kepler/lightcurves/0010/001026032/kplr001026032-2009166043257_llc.fits', cache=True)  # noqa
```

Comment:
This is from http://archive.stsci.edu/kepler/preview.php?dsn=KPLR001026032-2009166043257&type=LC that @fergalm provided.

Reply:
Does the remote_data() do caching? Otherwise we should have a conversation about downloading data within unit tests. It's not strictly halal from a TDD perspective, but tests against real data are quite nice otherwise.

Reply:
Caching for CI is not automatically done and needs special setup. From https://docs.travis-ci.com/user/caching/#arbitrary-directories , we will have to identify where Travis CI is storing `astropy` cache and then cache that directory.

Unless each server hit is expensive or the server is unstable, I don't see a big problem with a few remote data tests here and there.

Other options:

* Include this file as test data in the package. Given that it is like 200 KB, it is not too big, but things can easily get out of control if we start collecting a bunch of them.
* I can hardcode the arrays in the test code. Makes the file a bit harder to read though.

What do you think?

Reply:
Let's do what you're doing now, and just the issue in mind as we add more tests. We may decide to have a few example targets where we keep the data locally and use that data for all tests.